repl.lisp

```diff
@@ -334,7 +334,15 @@ strings to match candidates against (for example in the form \"package:sym\")."
     (unless text (end))
     (if (string= text "") (sbcli "" *prompt*))
     (when *hist-file* (update-hist-file text))
-    (handle-input txt text)
+    (cond
+      ((str:ends-with-p " ?" text)
+       ;; a trailing " ?" prints the symbol documentation.
+       (sbcli::symbol-documentation (str:trim
+                                     (str:replace-using (list "("  ""
```

Comment:
Why are we replaycing opening parens here?

Reply:
I want to be able to type "(pack:sym ?", but SYMBOL-FUNCTION only wants "pack:sym", so I strip the "(". Just in case, for convenience use in the REPL.

Reply:
I’m not sure this is good default behavior? Why for one `(`, but not deeply nested expressions, for instance?

Reply:
I didn't feel the need, but it should be OK in this last commit.

Reply:
It’s more for consistency than anything else TBH. I’ll look at it tonight!

Reply:
That's your role of code reviewer and you did good :]